blocks/init/src/Blocks/components/accordion/components/accordion-editor.js

```diff
@@ -43,7 +40,7 @@ export const AccordionEditor = (attributes) => {
 						<RichText
 							placeholder={placeholder}
 							value={accordionTitle}
-							onChange={(value) => setAttributes({ [`${componentName}Title`]: value })}
+							onChange={(value) => setAttributes({ [getAttrKey('accordionTitle', attributes, manifest)]: value })}
```

Comment:
How come you didn't use the `${componentName}Title` for the first argument in the `getAttrKey`?

Reply:
in general we can use it but we didn't use it in the checkAttr and this has the same props as checkAttr  and it is easier this way for the user to understand what is happening because the full attribute name corresponds to the name defined in the manifest under the attributes key. For me this looks cleaner